bin/generate-tests.php

```diff
@@ -11,4 +15,16 @@
     )
 );
 
-Controller::init()->run();
+(new Application('PHPUnit-Test-Generator', '1.0.0'))
+    ->register('generate-tests')
```

Comment:
please register a command class, so that this file here has as little logic as possible.

Reply:
and add defaults